mmrotate/datasets/dota.py

```diff
@@ -236,10 +236,16 @@ def merge_det(self, results):
             collector[oriname].append(new_result)
 
         merge_func = partial(_merge_func, CLASSES=self.CLASSES, iou_thr=0.1)
+        nproc = min(4, os.cpu_count())
+        if nproc <= 1:
```

Comment:
Can we set the `nproc` as an input of the `format_results` function so that it can be modified by users?

Reply:
The user may be confused as to how much this parameter is set appropriately.

Reply:
We need to use a single process to call `_merge_func` when deadlock happened. But `nproc = min(4, os.cpu_count())` is usually bigger than 1, which cannot make the program switch from `track_parallel_progress' to 'track_iter_progress'.

Reply:
Baeed on previous conventions, we should add an argument of `nproc` in this function. Usually users can set 4 and this could accelerate the process. The users can also set 1 to avoid parallel processing manually. We can add description in the docstring and documentation.

Reply:
OK